### PR TITLE
update mindyolo readme

### PR DESCRIPTION
**Pull request by WongGawa**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,20 @@
...
 MindYOLO implements state-of-the-art YOLO series algorithms based on MindSpore.
 The following is the corresponding `mindyolo` versions and supported `mindspore` versions.
-|  mindyolo  |  mindspore  |
-|    :--:    |     :--:    |
-|   master   |    master   |
-|    0.4     |    2.3.0    | 
-|    0.3     |    2.2.10   | 
-|    0.2     |    2.0      | 
-|    0.1     |    1.8      |
+|  mindyolo  |  mindspore  |  tested hardware
```
Comment (zhanghuiyao) [inline README.md:17]: 少了个 `|` 符号？

**Review by Ash-Lee233 — DISMISSED** [review]

File: examples/finetune_SHWD/README.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ optimizer:
 参数继承关系和参数说明可参考[configuration_CN.md](../../tutorials/configuration_CN.md)。
 
 #### 下载预训练模型
-可选用MindYOLO提供的[MODEL_ZOO](../../MODEL_ZOO.md)作为自定义数据集的预训练模型，预训练模型在COCO数据集上已经有较好的精度表现，相比从头训练，加载预训练模型一般会拥有更快的收敛速度以及更高的最终精度，并且大概率能避免初始化不当导致的梯度消失、梯度爆炸等问题。
```
Comment (zhanghuiyao) [inline examples/finetune_SHWD/README.md:103]: 最近合入的car_detection 的 readme也可以同步改下哈

File: README.md (hunk 1 above)
Comment (WongGawa) [reply README.md]: image
原先md也能实现一样结果，我新增一下。

File: examples/finetune_SHWD/README.md (hunk 2 above)
Comment (WongGawa) [reply examples/finetune_SHWD/README.md:103]: done

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**